tutorials/position-velocity-diagrams/PVDiagramPlotting.ipynb

```diff
@@ -0,0 +1,586 @@
+{
```

Comment:
<p>"First we create an extraction path" -- ????</p><p> </p><p>Do you mean region? A 3D region?</p><p> </p><p>Can you say "First, we need to create a region over which we will extract the PV information. This can be done using the <code>Path</code>  class provided by pvextractor"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/503/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
No, this is actually a 2D path, not a 3D region.  But the concept does deserve further explanation.

Reply:
I think this will be a lot easier to understand if you look at the rendered notebook